docs-devsite/vertexai.imagengenerationconfig.md

```diff
@@ -10,6 +10,13 @@ https://github.com/firebase/firebase-js-sdk
 {% endcomment %}
 
 # ImagenGenerationConfig interface
+> This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
+> 
+
+Configuration options for generating images with Imagen.
+
+See the [Cloud documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/image/generate-images) for more details.
```

Comment:
We'll have Firebase specific docs about this.

The URL will be http://firebase.google.com/docs/vertex-ai/generate-images

Note that the URL will 404 until launch day.